src/catalog.js

```diff
@@ -168,7 +161,13 @@ angular.module('gettext').factory('gettextCatalog', function (gettextPlurals, ge
             for (var key in strings) {
                 var val = strings[key];
 
-                if (isHTMLModified) {
+                // test for the presence of HTML tags in the key, as indicated
+                // by `<`
+                if (key.indexOf('<') !== -1) {
+                    // IE8 returns UPPER CASE tags, even though the source is lower case.
+                    // This can causes the (key) string in the DOM to have a different case to
+                    // the string in the `po` files.
+                    // IE9, IE10 and IE11 and recent FF verions reorder the attributes of tags.
                     // Use the DOM engine to render any HTML in the key (#131).
                     key = angular.element('<span>' + key + '</span>').html();
```

Comment:
@rubenv I was talking about this one, sorry for the confusion – I know it’s only during initialization.